CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
```

Comment:
@odscjames I'm planning on making a flatten-tool release on Monday, so that I can get the changes into CoVE.  The CHANGELOG will be updated when this PR is merged. Just thought I'd check that this looks good to you.

Reply:
Sounds good, but remember there is also the version number in setup.py to bump. 

I guess in this case you know nothing else will go in by then (so there will be no merge issues), but this might be something to think about in the future.